presto-docs/src/main/sphinx/release/release-0.130.rst

```diff
@@ -6,3 +6,7 @@ General Changes
 ---------------
 
 * Add :func:`map_concat` function.
+* Performance improvements for ``FilterAndProject`` operators. This includes projection
```

Comment:
FilterAndProject doesn't mean much from a user's point of view. Just say it's an optimization that improves performance for filters and projections, and for dictionary-encoded data.